test/unit/puppetlabs/services/jruby/jruby_puppet_service_test.clj

```diff
@@ -1,82 +1,31 @@
 (ns puppetlabs.services.jruby.jruby-puppet-service-test
-  (:import (com.puppetlabs.puppetserver JRubyPuppet))
   (:require [clojure.test :refer :all]
             [puppetlabs.services.protocols.jruby-puppet :as jruby-protocol]
-            [puppetlabs.services.jruby.jruby-testutils :as jruby-testutils]
+            [puppetlabs.services.jruby.jruby-puppet-testutils :as jruby-testutils]
             [puppetlabs.services.jruby.jruby-puppet-service :refer :all]
+            [puppetlabs.services.jruby-pool-manager.jruby-pool-manager-service :refer [jruby-pool-manager-service]]
             [puppetlabs.kitchensink.core :as ks]
             [puppetlabs.kitchensink.testutils :as ks-testutils]
             [puppetlabs.trapperkeeper.app :as app]
             [puppetlabs.trapperkeeper.core :as tk]
-            [puppetlabs.trapperkeeper.services :as services]
-            [clojure.stacktrace :as stacktrace]
+            [puppetlabs.services.jruby.jruby-puppet-core :as jruby-puppet-core]
             [puppetlabs.trapperkeeper.testutils.bootstrap :as bootstrap]
-            [puppetlabs.trapperkeeper.testutils.logging :as logging]
             [puppetlabs.services.puppet-profiler.puppet-profiler-service :as profiler]
-            [puppetlabs.services.jruby.jruby-puppet-core :as jruby-core]
-            [puppetlabs.services.jruby.jruby-puppet-internal :as jruby-internal]
-            [puppetlabs.services.jruby.jruby-puppet-schemas :as jruby-schemas]
             [me.raynes.fs :as fs]
             [schema.test :as schema-test]))
 
-(use-fixtures :each jruby-testutils/mock-pool-instance-fixture)
 (use-fixtures :once schema-test/validate-schemas)
 
 (defn jruby-service-test-config
   [pool-size]
   (jruby-testutils/jruby-puppet-tk-config
     (jruby-testutils/jruby-puppet-config {:max-active-instances pool-size})))
 
-(defn jruby-service-test-config-with-timeouts
-  [connect-timeout idle-timeout]
-  (merge (jruby-service-test-config 1)
-         {:http-client {:connect-timeout-milliseconds connect-timeout
-                        :idle-timeout-milliseconds    idle-timeout}}))
-
 (def default-services
   [jruby-puppet-pooled-service
+   jruby-pool-manager-service
    profiler/puppet-profiler-service])
 
-(deftest test-error-during-init
```

Comment:
Yeah, on second thought, I think I'm good with this test going away.  Seems like it would have been more worthwhile in the first place as a fuller-blown integration-y style test that includes real Puppet Server services that might encounter an error during JRuby initialization and bubble the error all the way up the stack appropriately.  But this test wasn't that before anyway and may not be worth the effort to change it to be that now.